docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md

```diff
@@ -12,7 +12,7 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
 
 > NOTE
 >
-> This syntax only supports StarRocks tables (`"ENGINE" = "OLAP"`). For Elasticsearch and Hive tables, use SHOW PROC '/dbs/db_id/table_id/partitions'.
+> This syntax only supports StarRocks tables (`"ENGINE" = "OLAP"`).
```

Comment:
Suggestion: This syntax only supports StarRocks tables (namely, tables for which `"ENGINE" = "OLAP"` is specified at table creation).